### PR TITLE
Fix overlap

### LINKED ISSUE #52: Grouped avatar overlap and mask offset are not as designed
The mask offset (called “margin” in the mockup) and overlap of grouped avatars is too wide.

Compare to the design for correct spacings (https://github.com/vaadin/vaadin-avatar/issues/8#issuecomment-648672806):

### Current implementation (incorrect):
Screenshot 2020-08-03 at 13 06 22

### Design: 

For the overlap, the abbreviations should be roughly equally visible regardless of the size variant. Currently, for example, the “FB” is being clipped too much in the smaller variants.

### PR DESCRIPTION
**Pull request by yuriy-fix**
Fixes #52

### CODE REVIEW

File: theme/lumo/vaadin-avatar-styles.html [hunk 1]
```diff
@@ -31,21 +31,24 @@
...
       [part="abbr"] {
         font-family: var(--lumo-font-family);
-        font-size: 3em;
+        font-size: 2.4375em;
         font-weight: 500;
       }
 
-      :host([theme~="xlarge"]) [part="abbr"],
+      :host([theme~="xlarge"]) [part="abbr"] {
+        font-size: 2.75em;
```
Comment (web-padawan) [inline theme/lumo/vaadin-avatar-styles.html]: Looks like `2.5em` would be better. Here is a pixel perfect match:

Screenshot 2020-08-18 at 14 08 59

File: theme/lumo/vaadin-avatar-styles.html [hunk 2]
```diff
@@ -31,21 +31,24 @@
...
+      }
+
       :host([theme~="large"]) [part="abbr"] {
-        font-size: 2.875em;
+        font-size: 2.4375em;
```
Comment (web-padawan) [inline theme/lumo/vaadin-avatar-styles.html]: Looks like `2.375em` would be a bit better here:

Screenshot 2020-08-18 at 14 10 43

**Review by web-padawan — CHANGES_REQUESTED** [review]
Two comments regarding font sizes for `xlarge` and `large`.

File: theme/lumo/vaadin-avatar-styles.html (hunk 1 above)
Comment (yuriy-fix) [reply theme/lumo/vaadin-avatar-styles.html]: Good point! Updated.

File: theme/lumo/vaadin-avatar-styles.html (hunk 2 above)
Comment (yuriy-fix) [reply theme/lumo/vaadin-avatar-styles.html]: Good point! Updated.

**Review by web-padawan — APPROVED** [review]

**Pull request merged**